packages/plugins/AddressSearch/README.md

```diff
@@ -26,14 +26,14 @@ In `categoryProperties` and `groupProperties`, id strings called `groupId` and `
 | - | - | - |
 | searchMethods | searchMethodsObject[] | Array of search method descriptions. Only searches configured here can be used. |
 | afterResultComponent | VueConstructor? | If given, this component will be rendered in the last line of every single search result. It will be forwarded its search result feature as prop `feature` of type `GeoJSON.Feature`, and the focus state of the result as prop `focus` of type `boolean`. |
-| addLoading | string? | Optional loading action name to start loading. |
+| addLoading | string? | Expects the path to an action within the store. This action is committed with a loading key as the payload when starting reverse geocoding. |
```

Comment:
This is not the reverse geocoder. I guess this is from a copy paste error. Also, actions can not be commited; only mutations can. Actions would require dispatchment.

I'd suggest the following change for a copyable field description that can be used in all places (with the replacement of "starting" with "finishing", where it fits).

```suggestion
| addLoading | string? | Expects the path to a mutation within the store. This mutation is committed with a plugin-specific loading key as payload when starting asynchronous procedures that are intended to be communicated to the user. |
```